app/models/review.rb

```diff
@@ -5,5 +5,4 @@ class Review < ApplicationRecord
   validates_presence_of :rating
   validates_presence_of :content
   validates_presence_of :shelter
-  validates_presence_of :picture
```

Comment:
The only way I could get the picture to be optional was to remove it from the validation. Everything works properly, so I am assuming this route is okay.

Reply:
Perfect